### DISCUSSION TITLE
SSL Certificate

### DISCUSSION THREAD

**Discussion by aa18afr** [Security-Onion-Solutions/securityonion#11253] [General] [answered]
Hi All,

I need to put SSL certificate. Can someone please suggest a way to do it?

I have tried to following methods:
> Replaced files in /etc/pki/managerssl.crt with my certificate file with same name as managerssl.crt but of no use. 

> Also, I placed my certs. in the /etc/ssl directory and provided path to nginx (/opt/so/saltstack/default/salt/nginx/etc/nginx.conf)

Please suggest a way forward, many thanks.

**Comment by eb-op** [comment, marked as answer]
Hi, aa18afr, what version of Security Onion are you currently running? There is some documentation on doing this in 2.4 through the Web UI, here: https://docs.securityonion.net/en/2.4/nginx.html#replacing-default-cert (.) If on 2.3, you will need to make sure your .crt and .key files are matching the same name as managerssl .crt and .key files. This is what Nginx is looking for. Once that has been completed, restart the Nginx service and ensure you are seeing the right certificate in your browser. Hope this helps!
